sdk/textanalytics/Azure.AI.TextAnalytics/tests/DetectLanguageTests.cs

```diff
@@ -249,5 +232,50 @@ public async Task DetectLanguageBatchWithNullTextTest()
             InvalidOperationException ex = Assert.Throws<InvalidOperationException>(() => results[0].PrimaryLanguage.GetType());
             Assert.AreEqual(exceptionMessage, ex.Message);
         }
+
+        private void ValidateInDocumenResult(DetectedLanguage language)
+        {
+            Assert.That(language.Name, Is.Not.Null.Or.Not.Empty);
```

Comment:
Should it be `Is.Not.Null.And.Not.Empty` instead? With `Or`, both null and empty will pass.

Reply:
updated